app/scripts/services/dimStoreService.factory.js

```diff
@@ -1107,6 +1108,10 @@
       })), 'sort');
     }
 
+    function isSaleItemUnlocked(saleItem) {
+      return _.every(saleItem.unlockStatuses, function(status) { return status.isSet; });
```

Comment:
Fun underscore trick:

```js
return _every(saleItem.unlockStatuses, 'isSet');
```

Not necessary to actually change it, but thought you might be amused to know about it.